new/LIT7153MiraclesY.xml

```diff
@@ -0,0 +1,355 @@
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ተአምሪሁ፡ ለአባ፡ ዮሐኒ፡ ዘደብረ፡ ዐሣ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Taʾāmrihu la-ʾAbbā Yoḥanni za-Dabra ʿAśā</title>
+                <title xml:lang="en" corresp="#t1">Miracles of ʾAbbā Yoḥanni of Dabra ʿAśā</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MV"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+            <abstract>
+                <p>
+                    This is a cycle of 24 miracles following <ref type="work" corresp="LIT7152GadlaY"/>.
+                    It is part of <ref type="work" corresp="LIT7156DossierYo"/>.
+                </p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                </keywords>
+            </textClass>         
+        </profileDesc>
+        <revisionDesc>
+            <change who="MV" when="2024-12-03">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">            
+                <listRelation>
+                    <relation name="ecrm:P129_is_about" active="LIT7153MiraclesY" passive="PRS10428Yohanni"/>
+                    <relation name="saws:formsPartOf" active="LIT7153MiraclesY" passive="LIT7156DossierYo"/>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <note>The following incipits are taken from MS Ethio-SPaRe ADMY-014 <!-- ms record missing -->.</note>
+                <div type="textpart" xml:id="MiracleArab">
+                    <label>First miracle of ʾAbbā Yoḥanni</label>
```

Comment:
since we cannot be 100% sure that the order and quantity are always the same I think it would be possibly more reasonable to use what you have as `note `as `label`